tests/framework/web/ControllerTest.php

```diff
@@ -80,7 +80,6 @@ public function testNullableInjectedActionParams()
                 ],
             ],
         ]));
-        $this->mockWebApplication(['controller' => $this->controller]);
```

Comment:
Interesting. What is the idea of this fix?

Reply:
within before method, the mockWebApplication already called and inside this test, it was called again and the __DIR__ cause some differences which make the tests fail.